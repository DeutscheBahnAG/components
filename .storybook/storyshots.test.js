import initStoryshots from '@storybook/addon-storyshots';

/* 
 * storyshots is a addon to storybook, which reads all stories and
 * automatically creates Jest snapshot tests for them, so we do not
 * need to do it manually ✌️
 * https://github.com/storybooks/storybook/tree/master/addons/storyshots
 * 
 * read about Jest's snapshot testing feature here:
 * https://facebook.github.io/jest/docs/en/snapshot-testing.html
*/

initStoryshots();