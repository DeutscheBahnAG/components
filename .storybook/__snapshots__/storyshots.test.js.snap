// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Basic Components/Button full width version 1`] = `
<button
  className="button button--block"
  onClick={[Function]}
  type="button"
>
  Full Width Button
</button>
`;

exports[`Storyshots Basic Components/Button with text 1`] = `
<button
  className="button"
  onClick={[Function]}
  type="button"
>
  Hello Button
</button>
`;
